update: refer users to the CCGP API reference instead of a manual list (#4695)
JetBrains/kotlin-web-site: docs/topics/compose-compiler-options.md

There are two kinds of options you can specify:

> * General compiler settings which are likely to be disabled or enabled as needed in any given project.

Reviewer: ```suggestion
* General compiler settings, which can be disabled or enabled as needed in any given project.
```

* Feature flags that enable or disable new and experimental features, which should eventually become part of the baseline.

You can find the [list of available general settings](https://kotlinlang.org/api/kotlin-gradle-plugin/compose-compiler-gradle-plugin/org.jetbrains.kotlin.compose.compiler.gradle/-compose-compiler-gradle-plugin-extension/)
and the [list of supported feature flags](https://kotlinlang.org/api/kotlin-gradle-plugin/compose-compiler-gradle-plugin/org.jetbrains.kotlin.compose.compiler.gradle/-compose-feature-flag/-companion/)

> in the Compose Compiler Gradle Plugin API reference.

Reviewer: ```suggestion
in the Compose compiler Gradle plugin API reference.
```

}
```

> The Gradle plugin provides defaults for several Compose compiler options that were only specified manually before Kotlin 2.0.

> > If you have any of them set up with `freeCompilerArgs`, for example, Gradle will report a duplicate options error.

Reviewer: ```suggestion
> If you have any of them set up with `freeCompilerArgs`, for example, Gradle reports a duplicate options error.
```
Removing present tense.

-P plugin:androidx.compose.compiler.plugins.kotlin:featureFlag=<flag name>
```

See the [list of supported feature flags](https://kotlinlang.org/api/kotlin-gradle-plugin/compose-compiler-gradle-plugin/org.jetbrains.kotlin.compose.compiler.gradle/-compose-feature-flag/-companion/)

> in the Compose Compiler Gradle Plugin API reference.

Reviewer: ```suggestion
in the Compose compiler Gradle plugin API reference.
```
